.github/workflows/test.yaml

```diff
@@ -109,7 +109,7 @@ jobs:
       # it as one of the sources.
       PYCTDEV_SELF_CHANNEL: "pyviz/label/dev"
     steps:
-      - uses: pyviz-dev/holoviz_tasks/install@v0.1a2
+      - uses: pyviz-dev/holoviz_tasks/install@v0.1a4
```

Comment:
Different versions?

Reply:
Good catch thanks that was an oversight. I wanted anyway to wait until you release 0.15.1 to make these changes since they're just meant to remove some deprecation warnings emitted by GitHub.